code/game/objects/items/stacks/stack.dm

```diff
@@ -12,21 +12,33 @@
 /obj/item/stack
```

Comment:
Ton of unrelated changes in this file

Reply:
Is making docs and refactoring the parent of the main subject of the thing I'm refactoring really going to hurt?